configure.ac

```diff
@@ -487,34 +487,35 @@ AS_IF([test x$have_gui = xyes],
 [
   # settings for Qt
   LIBS="$LIBS $QT_LIBS"
-  PKG_FLAGS="$PKG_FLAGS $QT_CFLAGS"
+  # add -fPIC flag to satisfy qt on linux
```

Comment:
The following line should make the comment obsolete, since it says exactly what we are trying to do here.